components/kyma-metrics-collector/README.md

```diff
@@ -0,0 +1,299 @@
+### Kyma Metrics Collector
```

Comment:
```suggestion
# Kyma Metrics Collector
```
Adjusting because of the Kyma README [template](https://github.com/kyma-project/community/blob/main/guidelines/repository-guidelines/repository-template/README.md).